assets/kube-apiserver/config.yaml

```diff
@@ -17,6 +17,17 @@ admission:
         restrictedCIDRs:
         - {{ .PodCIDR }}
         - {{ .ServiceCIDR }}
+    PodSecurity:
+      configuration:
+        kind: PodSecurityConfiguration
+        apiVersion: pod-security.admission.config.k8s.io/v1alpha1
```

Comment:
Please use `apiVersion: pod-security.admission.config.k8s.io/v1beta1` and make other updates to match https://github.com/openshift/cluster-kube-apiserver-operator/blob/release-4.11/bindata/assets/config/defaultconfig.yaml#L12-L22

Reply:
I'll have to get back to this later.  Trying to set up new machine.

Reply:
Bah, so basically the entire content is different and it didn't register.  Should be right now.